EchoServer.UI/Program.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace EchoServer
```

Comment:
Am I to believe that the EchoServer is a separate project from the HTTP Server?  In other words, this will be a stand alone project and when it comes time to work on an HTTP server you will create a new repo?

Reply:
Well, I named it EchoServer because I thought that was the first step toward the HTTP and I did not know if they would be separate files/projects or one and the same.  It seems that they are one and the same...is that correct?

Reply:
The project you are working on is an HTTP server.

Reply:
an Echo server is an HTTP server that does one thing .. excepts a POST request, with a `body`, and echos that `body`.  It is in essence the simplest thing that you can write first.

Reply:
once you have an echo server, you can then begin to add full POST support for other methods (GET, PUT, DELETE, etc ...).

Reply:
Got it!  I would like to change it if it will be easier to just build onto the Echo Server in one project, but I don't want to mess anything up if I rename the repo and then rename the files.  I don't think I've ever done that before so if you have any suggestions or assurances it's an easy fix, I can do that now.

Reply:
@t-keazirian I see you did the rename.  However, dotnet already has an HTTP library and this name could conflict with it.  I highly suggest you add something unique to the name as suggested above.

Alternatively, you could may it fun.  something like `Servy` or `TKHTTP`.

Reply:
I think the pattern is `something to identify the author` . `the package name`